ush/templates/FV3LAM_wflow.xml

```diff
@@ -105,14 +105,16 @@ tasks; and the "FCST" type is used for the RUN_FCST_TN task.
 
 ]>
 
-<workflow realtime="T" scheduler="&SCHED;" cyclethrottle="3" cyclelifespan="01:00:00:00">
+<workflow realtime="T" scheduler="&SCHED;" cyclethrottle="25" cyclelifespan="01:00:00:00">
 {# Double quotes are required inside the strftime! Expect an error from reading the template if using single quotes. #}
+{%- if (run_task_make_grid or run_task_make_orog or run_task_make_sfc_climo) %}
   <cycledef group="at_start">{{ cdate_first_cycl.strftime("%M %H %d %m %Y *") }}</cycledef>
-  <cycledef group="forecast">{{ cdate_first_cycl.strftime("%Y%m%d%H%M") }} {{ cdate_last_cycl.strftime("%Y%m%d%H%M") }} {{ cycl_freq }}</cycledef>
+{% endif %}
+  <cycledef group="forecast">00 {% for cycl_hr in cycl_hrs[:-1] -%} {{ cycl_hr }},{%- endfor -%}{{ cycl_hrs[-1] }} * * {{ cdate_first_cycl.strftime("%Y") }}-{{ (cdate_last_cycl.strftime("%Y")|int)+1 }} *</cycledef>
 {%- if do_dacycle %}
-  <cycledef group="dacycle">00 03,06,09,15,18,21 {{cdate_first_cycl.strftime("%d") }}-{{ cdate_last_cycl.strftime("%d") }} {{ cdate_first_cycl.strftime("%m") }} {{cdate_first_cycl.strftime("%Y") }} *</cycledef>
+  <cycledef group="dacycle">00 {% for dacycl_hr in dacycl_hrs[:-1] -%} {{ dacycl_hr }},{%- endfor -%}{{ dacycl_hrs[-1] }} * * {{ cdate_first_cycl.strftime("%Y") }}-{{ (cdate_last_cycl.strftime("%Y")|int)+1 }} *</cycledef>
 {% endif %}
-  <cycledef group="archive">{{ cdate_first_arch.strftime("%Y%m%d%H%M") }} {{ cdate_last_arch.strftime("%Y%m%d%H%M") }} 24:00:00</cycledef> 
+  <cycledef group="archive">00 {{ cdate_first_arch.strftime("%H") }} * * {{ cdate_first_cycl.strftime("%Y") }}-{{ (cdate_last_cycl.strftime("%Y")|int)+1 }} *</cycledef> 
```

Comment:
What happens to the Rocoto database with this notation? Will it start on Jan 1, 07 UTC? 

If we make this change, it will get us further away from easily supporting retros with similar notation. Specifying start, stop, and interval where we can may be helpful for enabling retro runs.

Reply:
Yes the database will start on Jan 1st.  But all of the cycles before the current day will be inactive.  How do you specify an interval if the interval is irregular?

Reply:
Note I changed the "archive" cycledef here not because it was necessary, but just to be consistent with the other cycledefs

Reply:
What is the goal of this change? The previous way (red line) has a start, stop, and interval specified, and you're right, the interval is set to be consistent. 

Could we leave the start, stop, interval method in place for all the ones that we can, and only do the cron method for the ones where we don't have a set interval? I think it's generally easier for retros that way. And for managing the real time runs when we start on a given day.

Reply:
That would make archive cycle definitions active for different days than the forecast cycle definitions.  Depends how you feel about that.  That allows for a situation where forecasts keep going but don't get archived.  It might be better to have everything expire at once.  How about a retro/real-time switch in config.sh, so we don't have to keep making compromises to achieve both goals?